app/src/main/java/com/infomaniak/mail/workers/DraftsActionsWorker.kt

```diff
@@ -138,8 +138,9 @@ class DraftsActionsWorker @AssistedInject constructor(
         val errorMessageResIds = mutableListOf<Int>()
         var remoteUuidOfTrackedDraft: String? = null
         var trackedDraftAction: DraftAction? = null
+        var isTrackedDraftSuccess = false
 
-        val isSuccess = mailboxContentRealm.writeBlocking {
+        val isAllDraftSucceeded = mailboxContentRealm.writeBlocking {
```

Comment:
`haveAllDraftsSucceeded` or at the very least `allDraftsSucceeded` but`isAllDraftSucceded` is grammaticaly very wrong